packages/maps/src/components/result/ReactiveGoogleMap.js

```diff
@@ -242,6 +243,7 @@ ReactiveGoogleMap.defaultProps = {
 	showMarkerClusters: true,
 	unit: 'mi',
 	defaultRadius: 100,
+	mapService: 'GoogleMap',
```

Comment:
This prop is internal to `ReactiveMap` and shouldn't be exposed to end-user. There are more chances of user misusing it. example

`<ReactiveGoogleMap mapService="OpenStreentMap" ... />` and this will break the maps.